package.json

```diff
@@ -26,6 +26,15 @@
       "require": "./dist/open-props.cjs",
       "default": "./dist/open-props.cjs"
     },
+    "./src": "./src/index.js",
+    "./src/props.animations.js": "./src/props.animations.js",
+    "./src/props.aspects.js": "./src/props.aspects.js",
+    "./src/props.borders.js": "./src/props.borders.js",
+    "./src/props.colors.js": "./src/props.colors.js",
+    "./src/props.colors-hsl.js": "./src/props.colors-hsl.js",
+    "./src/props.easing.js": "./src/props.easing.js",
+    "./src/props.fonts.js": "./src/props.fonts.js",
+    "./src/props.gradients.js": "./src/props.gradients.js",
```

Comment:
missing sizes, shadows, and maybe something else? 
(i didnt do a full scrub)

Reply:
Yes. I stopped scrolling too early.